test/inductor/test_cpu_repro.py

```diff
@@ -3657,6 +3657,30 @@ def fn(x):
         self.common(fn, (x,))
         assert metrics.generated_cpp_vec_kernel_count == 1
 
+    def test_issue_128263(self):
```

Comment:
maybe you want to change this to a more descriptive name?

Reply:
Thanks, changed to a more descriptive name.